common/css/devdocs.css

```diff
@@ -19,4 +20,7 @@
 
 .ref-list dt{
   font-weight:500;
+}
+.versions {
```

Comment:
Ideally, i would recommend to have more descriptive class name (`versions` is too generic).

Reply:
You can be more specific by using this selector:
```
.message-banner.versions
```

Reply:
Done.